packages/processors/src/helpers/pricing.ts

```diff
@@ -0,0 +1,71 @@
+import { IPricingProvider } from "@grants-stack-indexer/pricing";
+import { Token } from "@grants-stack-indexer/shared";
+
+import { TokenPriceNotFoundError } from "../internal.js";
+import { calculateAmountInToken, calculateAmountInUsd } from "./tokenMath.js";
```

Comment:
```suggestion
import { calculateAmountInToken, calculateAmountInUsd } from "./index.js";
```

Reply:
how to make autoimport work magically with this? maybe it's easier to have one single file called `index.js` with all the code ?)

Reply:
i know that is annoying but, it save us some future headaches 🤣